### DISCUSSION TITLE
New release on npm?

### DISCUSSION THREAD

**Discussion by SanteeMarcel** [ramda/ramda#3309] [General]
Is there a planned date for new releases? I'm looking forward for the fix for issue #3258

**Comment by adispring** [comment]
@CrossEye @customcommander @davidchambers 

Long time since last release(0.28.2), there are some new bugfixes and new features need to release, but it seems that ramda has been fell into an inactive status, such as fewer ramda members response to issues and prs, no new release, and so on.

So should we make ramda active again?
